frontend_lib/src/component/Timeline/Comment.jsx

```diff
@@ -71,12 +75,26 @@ const Comment = props => {
                       {props.author.public_name}
                     </span>
                   </ProfileNavigation>
-
                   <div
                     className={classnames(`${props.customClass}__body__content__header__meta__date`, 'comment__body__content__header__meta__date')}
-                    title={createdFormated}
                   >
-                    {createdDistance}
+                    <span id={`createdDistance_${props.apiContent.content_id}`}>
+                      {createdDistance}
+                    </span>
+                    <Popover
+                      targetId={`createdDistance_${props.apiContent.content_id}`}
+                      popoverBody={readableCreatedDate}
+                    />
+                    {isModified && (
+                      // Doesn't displayed by TLM; require a refresh to be displayed
```

Comment:
Forgotten comment?

Reply:
Solved in [fd85e03](https://github.com/tracim/tracim/pull/5846/commits/fd85e03f05599f33b4ee4f2e616fd52f78605da7).